woocommerce/woocommerce-android
# Display SKU only if it is not empty

This tiny PR fixes #3579 by hiding the SKU section in order detail screen, if the SKU is empty.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/108154387-ce4da380-7102-11eb-95f5-cb692efe46c2.png" width="300"/>

### Testing
- Click on an order with any number of products.
- Notice the SKU is only displayed if the product SKU is not empty.

Update release notes:
